assets/src/dashboard/app/views/editorSettings/publisherLogo/index.js

```diff
@@ -199,6 +178,18 @@ function PublisherLogoSettings({
                 return null;
               }
 
+              const items = [
```

Comment:
If we pulled `GridItemContainer` into a new component then this `items` array could be set there and tighten up readability a bit. I'm also curious if a custom hook to render these items would be helpful? 

Just spitballing. I want to avoid adding complexity.